src/script/assets/AssetCrypto.ts

```diff
@@ -17,13 +17,11 @@
  *
  */
 
-/**
- * @param {ArrayBuffer} cipherText - Encrypted plaintext
- * @param {ArrayBuffer} keyBytes - AES key used for encryption
- * @param {ArrayBuffer} referenceSha256 - SHA-256 checksum of the cipherText
- * @returns {Promise} Resolves with the decrypted asset
- */
-export const decryptAesAsset = (cipherText, keyBytes, referenceSha256) => {
+export const decryptAesAsset = async (
```

Comment:
Please do not remove the explanations.

Reply:
You mean the comment? Which value does it provide?

Reply:
The descriptions for the `keyBytes` and `referenceSha256` are helpful IMO. The other ones are not needed.

Reply:
Ok! I will bring them back. 👍